corehq/toggles.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, slug, label, tag, namespaces=None, help_link=None,
         # updated.  This is only applicable to domain toggles.  It must accept
         # two parameters, `domain_name` and `toggle_is_enabled`
         self.save_fn = save_fn
+        self.force_enable = force_enable
```

Comment:
I believe we've done this in the past by changing the toggle to a `PredictablyRandomToggle` and setting randomness to 1, so you wouldn't need to add another option.